packages/connect-explorer/package.json

```diff
@@ -1,6 +1,5 @@
 {
     "name": "@trezor/connect-explorer",
-    "packageManager": "yarn@4.0.2",
```

Comment:
This shoudn't be here and it breaks Nx sometimes...

Reply:
i am not sure why we have it there. Is it intended? @martykan

Reply:
Probably a leftover from the Nextra template